rllib/utils/serialization.py

```diff
@@ -325,3 +327,64 @@ def check_if_args_kwargs_serializable(args: Sequence[Any], kwargs: Dict[str, Any
                 f"Found non-serializable keyword argument: {k} = {v}.\n"
                 f"Original serialization error: {e}"
             )
+
+
+@DeveloperAPI
+def serialize_type(type_: Union[Type, str]) -> str:
+    """Converts a type into its full classpath ([module file] + "." + [class name]).
+
+    Args:
+        type_: The type to convert.
+
+    Returns:
+        The full classpath of the given type, e.g. "ray.rllib.algorithms.ppo.PPOConfig".
+    """
+    # Already serialized.
+    if isinstance(type_, str):
+        return type_
+
+    return type_.__module__ + "." + type_.__name__
```

Comment:
use `type.__qualname__` instead of `type_.__name__`

Reply:
done